lexicons/com/atproto/server/createAppPassword.json

```diff
@@ -14,6 +14,10 @@
             "name": {
               "type": "string",
               "description": "A short name for the App Password, to help distinguish them."
+            },
+            "scopes": {
+              "type": "array",
+              "items": { "type": "string" }
```

Comment:
Might recommend eventually putting some basic max length limit here, even if it's high like `100`.

Reply:
oh oops this wasn't supposed to make it in there 🤦